src/background.js

```diff
@@ -56,6 +60,8 @@ function setupNotification(timer) {
     chrome.storage.local.get({soundType: "tts", soundId: "ring"}, function(object) {
       if (object.soundType == "tts") {
         chrome.tts.speak(timer.desc);
+      } else if (object.soundType == "mute") {
```

Comment:
How about not doing anything when soundType is mute? Then the code would be like

```
} else if (object.soundType == "ring") {
    audios[object.soundId].play();
}
// nothing for mute case
```

In this case, noSound.wav does not have to be included.